tests/postgres_tests/test_indexes.py

```diff
@@ -119,3 +138,24 @@ def test_brin_index(self):
         with connection.schema_editor() as editor:
             editor.remove_index(CharFieldModel, index)
         self.assertNotIn(index_name, self.get_constraints(CharFieldModel._meta.db_table))
+
+    @skipUnlessDBFeature('has_gin_index_pending_list_parameter')
+    def test_gin_parameters(self):
+        # Ensure the table is there and doesn't have an index.
+        self.assertNotIn('field', self.get_constraints(IntegerArrayModel._meta.db_table))
+        index_name = 'integer_array_model_field_gin'
+        index = GinIndex(
+            fields=['field'], name=index_name,
+            fastupdate=True, gin_pending_list_limit=64,
```

Comment:
Would it be better to (also?) test the parameters separately? Otherwise we have no testing for `fastupdate` on PostgreSQL < 9.5.